code/game/objects/items/ego_weapons/he.dm

```diff
@@ -177,8 +182,11 @@
 	if (block == 0)
 		var/mob/living/carbon/human/cooler_user = user
 		naked_parry = isnull(cooler_user.get_item_by_slot(ITEM_SLOT_OCLOTHING))
-		if(naked_parry)
-			reductions = list(95, 95, 95, 100) // Must be wearing 0 armor
+		realized_parry = user.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```

Comment:
Now this doesn't even check for the realized armor? Just if they're wearing any?